collector/config.yml

```diff
@@ -34,7 +34,7 @@ metrics:
 
 prometheus:
   # the exposed port for prometheus exposer
-  exposer_port : 1111 
+  exposer_port : 8080 
```

Comment:
Same, and 34*01 ports are used by Pegasus by habit, how about use 34201 instead?